README.md

```diff
@@ -0,0 +1,28 @@
+# COS Alerter
+
```

Comment:
**suggestion:** We should probably elaborate on the operational approach we're taking here, even if it may seem obvious.

For example:
```suggestion
COS Alerter is intended to be used together with alertmanager and prometheus:
- Liveness of Alertmanager from an always-firing alert rule ("Watchdog")
- Liveness of COS Alerter itself from a metric endpoint it exposes and prometheus scrapes
```

Reply:
i'd switch `from` to `through`, but other than that i agree with leon.